doc/catalog-format-spec.md

```diff
@@ -856,11 +856,13 @@ available.'case(toolchain)'.user = false
 
 ## Parameters
 
- - `os`: name of the OS. Currently supported values are: `linux`, `macos` and
-   `windows`.
+ - `os`: name of the OS. Currently supported values are: `freebsd`, `linux`,
+   `macos`, `os-unknown`, and `windows`.
 
- - `distribution`: name of the Linux distribution, or `none` if running on a
-   different OS. Currently supported values are: `debian`, `ubuntu`.
+ - `distribution`: name of the Linux distribution or name of the software
+   distribution platform if running on a different OS. Currently supported
+   values are: `arch`, `centos`, `debian`, `distro-unknown`, `fedora`,
+   `homebrew`, `macports`, `msys2`, `rhel`, `suse`, and `ubuntu`.
```

Comment:
This makes me realize that `distribution-unknown` would be more coherent. Let me get back to you once that's done.